mozdownload/scraper.py

```diff
@@ -36,7 +36,10 @@
 APPLICATIONS = ('b2g', 'firefox', 'thunderbird')
 
 # Base URL for the path to all builds
-BASE_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
+BASE_URL = 'http://download.cdn.mozilla.net/pub'
+
+# In case CDN fails
+BACKUP_URL = 'https://ftp.mozilla.org/pub/mozilla.org'
```

Comment:
I think you'll hit cert issues using https, at least I do loading your BASE_URL in Firefox and getting Akamai. We have an https CDN but it only has installers on it.